src/mail-app/settings/SettingsView.ts

```diff
@@ -573,7 +555,7 @@ export class SettingsView extends BaseTopLevelView implements TopLevelView<Setti
 		)
 	}
 
-	_getCurrentViewer(): Component {
+	_getCurrentViewer(): UpdatableSettingsViewer | null {
```

Comment:
why does this one return nullable result? shouldn't viewCreator give us a viewer?